homeassistant/components/vodafone_station/coordinator.py

```diff
@@ -39,6 +42,8 @@ class UpdateCoordinatorDataType:
 class VodafoneStationRouter(DataUpdateCoordinator[UpdateCoordinatorDataType]):
     """Queries router running Vodafone Station firmware."""
 
+    config_entry: ConfigEntry
```

Comment:
In a future PR it's a bit cleaner to explicitly pass the config entry

Reply:
#137276